### PR TITLE
docs: fix typo in MAPPING.adoc

### PR DESCRIPTION
**Pull request by ediel-queiroz**
Changes:
- Fix typo in MAPPING.adoc documentation file.

### CODE REVIEW

**Comment by genie-jnosql** [comment]
Can one of the admins verify this patch?

File: MAPPING.adoc [hunk 1]
```diff
@@ -224,13 +224,13 @@ public class Animal {
...
 
-Notice that the `Animal` doest not have an @Entity annotation, as it won't be persisted in the database by itself.
+Notice that the `Animal` doesn't have an @Entity annotation, as it won't be persisted in the database by itself.
 
 On the example above, when saving a Dog instance, Animal class' fields are saved too: `name`, `race`, and `age` are saved in a single instance.
 
 ===== @Inheritance
 
-The strategy to work with inheritance with NoSQL, you can active it by adding the @Inheritance annotation to the superclass.
+The strategy to work with inheritance with NoSQL. You can activate it by adding the @Inheritance annotation to the superclass.
```
Comment (dearrudam) [inline MAPPING.adoc:233]:
```suggestion
The strategy to work with inheritance with NoSQL can be activated by adding the `@Inheritance` annotation to the superclass.
```

**Review by dearrudam — CHANGES_REQUESTED** [review]

File: MAPPING.adoc (hunk 1 above)
Comment (ediel-queiroz) [reply MAPPING.adoc]: > 

Thanks for the suggestion, @dearrudam . Addressed in [e2db5c62c14938f757f05f0f95ee1c875ba9e722](https://github.com/eclipse/jnosql/pull/475/commits/e2db5c62c14938f757f05f0f95ee1c875ba9e722)

**Comment by dearrudam** [comment]
@ediel-queiroz thanks for the contribution!!!

**Pull request merged**